core/domain/stats_domain_test.py

```diff
@@ -547,6 +720,106 @@ def test_validate(self):
             'Invalid action type: %s' % 'InvalidActionType')):
             playthrough.validate()
 
+    def test_validate_exp_id(self):
```

Comment:
Please rename to: `test_validate_non_str_exp_id`

Reply:
done